suite-native/module-accounts-import/src/accountsImportSlice.ts

```diff
@@ -0,0 +1,39 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+import { networks } from '@suite-common/wallet-config';
+
+import { AssetItem } from './components/SelectableAssetList';
+
+export const actionPrefix = '@accountsImport';
+
+export type AccountsImportState = {
+    selectedNetwork: AssetItem;
+};
+
+export type AccountsImportRootState = {
+    accountsImport: AccountsImportState;
+};
+
+const initialState: AccountsImportState = {
+    selectedNetwork: {
+        cryptoCurrencySymbol: 'btc',
+        cryptoCurrencyName: networks.btc.name,
```

Comment:
You don't need to store anything, but symbol and then you can do `networks[symbol]` and you will get all of these properties. If you want to do this correctly you can do this in `selector`.

Anyway get rid of redux completelly just pass `networkSymbol` as navigation param, that should be enough and it will be even more versatile (for example when we will want to skip initial network selection screen, when we offer import of BTC account somewhere later in app)

Reply:
b8f267e3aaa51e650b8a3c97d743e402f351cae0